packages/inventory-compliance/src/Compliance.js

```diff
@@ -144,6 +146,13 @@ SystemDetails.propTypes = {
             inventoryId: propTypes.string
         })
     }),
+    columns: propTypes.shape([
+        {
+            title: propTypes.oneOfType([ propTypes.string, propTypes.object ]).isRequired,
```

Comment:
If the title should also accept React elements you could use `PropTypes.element`, instead of the `propTypes.oneOfType(...)`, which allows (i think both) strings and objects/elements.